### ISSUE TITLE
Multi-factor authentication

### ISSUE THREAD

**Issue by momorgan** [leavemealone-app/roadmap#47]
MFA. That's it. That's the issue.

**Comment by Jivings** [comment]
Does it not work? You should be able to set up from the Security page here: https://leavemealone.app/app/profile/security

**Comment by momorgan** [comment]
Aaah okay I didn't look hard enough then. Sorry. 

See now you've said that I don't know why I couldn't find it!

**Comment by Jivings** [comment]
No worries!

**Comment by momorgan** [comment]
I shall try and redeem my covid-ridden poorsightedness with two bits of low-priority feedback:

1. Icon picked up by Authy didn't match LMA logo - it has a keep-calmageddon thing going on. Might spook a user.

2. Perhaps at some point in the middle distance the settings could be searchable?

**Comment by Jivings** [comment]
Authy seems to suck at icons. I don't know if there's any way for me to change it is there?

**Comment by momorgan** [comment]
So I had a rummage around. My guess is they take some search terms (app name, initially?) and do a image API search (or scrape). The real pisser with Authy's logo search is it returns exactly one image for a search term.

Found this tweet from Authy that says "Our logo search feature relies on (and is restricted by) Google’s image search. We have had luck in toying w/  search terms in the tool, sometimes adding “new logo” after a brand name search yields different results."
https://twitter.com/authy/status/1235964094026600448?lang=en-GB

So I jigged around with it a little and yes, I can get different images but not the right image. I thought Google's Image API was long dead, and I also don't get the same results from an image search through the browser (typical google) so maybe it just comes down to SEO, sadly. The app comes up as 'LeaveMeAlone' so maybe 'leavemealone logo' and similar is what needs to be googlewhacked. One of the terms I tried returned one of the images from the blog (fastmail -> lma) so perhaps that's the place to do the hard optimisation. Dunno.
